.env.example

```
@@ -1,35 +1,18 @@
+# --- Django settings ---
+PROJECT_NAME=dj-site-templ
+DEBUG=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEBUG key should go before the PROJECT_NAME key

(UnorderedKey)